nomium/prometheus/README.md

```diff
@@ -0,0 +1,20 @@
+```
```

Comment:
Ensure metric descriptions are clear for all users, including non-Russian speakers.